sklearn/decomposition/tests/test_online_lda.py

```diff
@@ -202,7 +187,7 @@ def test_lda_transform_mismatch():
                                     random_state=rng)
     lda.partial_fit(X)
     assert_raises_regexp(ValueError, r"^The provided data has",
-                         lda.partial_fit, X_2)
+                         lda.transform, X_2)
```

Comment:
I changed that because as is, it was a duplicate of `test_lda_partial_fit_dim_mismatch` and I think i did not do what it was expected to do, i.e. "# test `n_features` mismatch in partial_fit and transform"